.changeset/sour-guests-sneeze.md

```diff
@@ -0,0 +1,6 @@
+---
+"@empiricalrun/ai": minor
+"web": minor
```

Comment:
shouldnt this be a patch ? given its a fix

Reply:
Updated both to `patch`